src/autopas/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ target_compile_definitions(
     PUBLIC
     $<$<BOOL:${AUTOPAS_OPENMP}>:AUTOPAS_OPENMP>
     $<$<BOOL:${AUTOPAS_ENABLE_CUDA}>:AUTOPAS_CUDA>
+    $<$<NOT:$<BOOL:${AUTOPAS_OPENMP}>>:EIGEN_DONT_PARALLELIZE>
```

Comment:
I think so too but without setting this flag the test on Jenkings failed. If I remember correctly "clang ninja address sanitizer" and "clang ninja address sanitizer release" could not compile because of missing OpenMP functions.